src/s3fs.cpp

```diff
@@ -149,7 +149,7 @@ static int set_bucket(const char* arg);
 static int my_fuse_opt_proc(void* data, const char* arg, int key, struct fuse_args* outargs);
 static fsblkcnt_t parse_bucket_size(char* value);
 static bool is_cmd_exists(const std::string& command);
-static int print_umount_message(const std::string& mp, bool force);
+static int print_umount_message(const std::string& mp, bool force) __attribute__ ((unused));
```

Comment:
In the case of `__MSYS__` it is `unused`, but in other cases this is used.
For now it's fine as it is, but it may be better to use ifdef(etc) on `__MSYS__` in the future. (However, this will make the code harder to read.)